docs/install/install-cave.md

```diff
@@ -208,3 +255,17 @@ sudo yum remove awips2-*
 rm -rf /awips2/cave
 rm -rf ~/caveData
 ```
+
+### Windows
+
+To completely remove CAVE:
+
+1. Type "remove" in the search bar and select the first option that comes up.  This will open the Applications settings.
```

Comment:
I would say:
Type "remove" in the search bar and select "Add or remove programs" because if the user has a local file named remove that could come up first...and I confirmed on my Windows 10 VM and Windows 11 it was the same